coderkevin/fresh-data
# Update examples package.json and lockfiles

This updates the package.json files to no longer require npm link for
the example code, just a good build before doing `npm install` on the
examples.

It also updates the package-lock.json files while we're at it.

To Test:
1. `npm install`
2. `npm run build`
3. `cd examples/hello-world`
4. `npm install`
5. `npm start` and ensure the app comes up.
6. `cd ../wp-rest-api`
7. `npm install`
8. `npm start` and ensure the app comes up.
